runtime/queries/haskell/highlights.scm

```diff
@@ -4,66 +4,37 @@
 ; and don't override destructured parameters
 (variable) @variable
 
-(pat_wildcard) @variable
+(pattern/wildcard) @variable
```

Comment:
I don't think Neovim can deal with these; it's the first time I've seen nodes using slashes. I would strongly recommend sticking to word-like characters.

Reply:
(Protip: `:TSUpdate` the parser, then open the query in Neovim; the query linter will tell you exactly what it doesn't like.)

Reply:
Hmm, interesting. 
I get the lint warnings, but highlighting seems to be working fine.
Those are `<super>/<node>` queries. 
I'll see if I can find an alternative in a few days. 

(cc @tek)

Reply:
Interesting. If it's just the tests (and query linter) that are affected, then maybe we can fix this. Still, sticking to the general snake_case convention would be preferable.

Reply:
(It doesn't error, but I don't think it's working correctly, either.)